src/js/controller/model.js

```diff
@@ -102,7 +102,7 @@ define([
 
                     /* falls through */
                 case events.JWPLAYER_MEDIA_META:
-                    var duration = data.duration;
+                    var duration = data.duration || 0;
```

Comment:
@hghazzi would it be safer, then, to remove `|| 0` and instead check the following before updating the duration in the model?
 `if (_.isNumber(duration) && !_.isNaN(duration)) {`

We shouldn't get `NaN` from the HTML5 or Flash providers anymore. But, if we do, we wont make unnecessary updates to the model that don't reflect the actual duration of the media being played.